azure-pipelines/build.yml

```diff
@@ -23,6 +23,8 @@ trigger:
   branches:
     include:
     - refs/heads/main
+  pr:
+    - main
```

Comment:
This pipeline builds stuff but it doesn't seem to run any tests or am I missing something?